scripts/ci/router-check.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+set -u
+set -o pipefail
+
+# wait for the router
+#KUBECONFIG="${HOME}"/clusters/nested/auth/kubeconfig
+KUBECONFIG="${HOME}"/Documents/installer/somalley/auth/kubeconfig
+API_UP=
+ROUTER_NAMESPACE=
+ROUTER_DEPLOYMENT=
+
+find_router_deployment()
+{
+  while true; do
+    if [[ ! -f "${KUBECONFIG}" ]]; then
+      sleep 15 & wait
+      continue
+    fi
+    if ! oc get nodes 2>/dev/null; then
+      echo "Waiting for API at $(oc whoami --show-server) to respond ..."
+      sleep 15 & wait
+      continue
+    elif [[ -z "${API_UP}" ]]; then
+      echo "API at $(oc whoami --show-server) has responded"
+      API_UP=1
+    fi
+    if [[ -z "${ROUTER_NAMESPACE}" ]]; then
+      # check multiple namespaces while we are transitioning to the new locations
+      if oc get deploy/router -n tectonic-ingress 2>/dev/null; then
```

Comment:
We can probably drop this case with #600 landed :tada:

Reply:
:+1:  finally!